frontends/ol-components/src/components/NavDrawer/NavDrawer.tsx

```diff
@@ -1,15 +1,19 @@
 import Drawer, { DrawerProps } from "@mui/material/Drawer"
+import ClickAwayListener from "@mui/material/ClickAwayListener"
+import { FocusTrap } from "@mui/base/FocusTrap"
 import styled from "@emotion/styled"
 import React, { ReactElement } from "react"
+import { RiCloseLargeLine } from "@remixicon/react"
+import { ActionButton } from "../Button/Button"
 
 const DrawerContent = styled.div(({ theme }) => ({
-  paddingTop: "72px",
+  paddingTop: theme.custom.dimensions.headerHeight,
   width: "366px",
   height: "100%",
   background: theme.custom.colors.white,
   borderRight: `1px solid ${theme.custom.colors.lightGray2}`,
   [theme.breakpoints.down("sm")]: {
-    height: "60px",
+    paddingTop: theme.custom.dimensions.headerHeightSm,
```

Comment:
The nav menu had extra top padding at mobile widths... I think the `height: 60px ` was supposed to be `paddingTop: 60px` all along. Changed to theme constants.